ebookstore-ui/src/RegistrationForm.tsx

```diff
@@ -66,9 +66,13 @@ export default function SignUp() {
                 },
                 body: JSON.stringify(object),
             });
-            if (response.status === 200) {
+            
+              if (response.status === 200) {
                 console.log('User registered successfully');
                 navigate('/signin');
+            } else if(response.status === 409){
+                setError("Unable to Register due to the duplciate credentials. Please try again with unique credentials")
```

Comment:
The error might be ambiguous to the user..
1. If the email is same - "User already exists. Try logging In"
2. If the Username has a conflict - "Username already exists, please pick a different one"

Reply:
We might have to send the details in body along with 409 status so that specific error can be printed out.

Reply:
Fixed it. Good suggestion. This will help the users to know what exactly going wrong when registering to our application.